spec/lib/code_ownership_spec.rb

```diff
@@ -66,6 +66,40 @@
         end
       end
     end
+
+    context 'file is unowned' do
+      before do
+        write_file('config/teams/bar.yml', <<~CONTENTS)
+          name: Bar
+        CONTENTS
+
+        write_configuration
+
+        write_file('app/services/autogenerated_code/some_unowned_file.rb', '')
+      end
+
+      it 'has no validation errors' do
```

Comment:
I wrote this test as a control to make sure normally the file WOULD have a validation error (so the following test is not a false positive).